server/util/db/db.go

```diff
@@ -51,3 +57,10 @@ func GetConfiguredDatabase(c *config.Configurator) (*DBHandle, error) {
 	}
 	return nil, fmt.Errorf("No database configured -- please specify at least one in the config")
 }
+
+func (d *DBHandle) StartOfDayTimestamp(offset int) string {
```

Comment:
Could we call this offsetDays or offsetPastDays or something cleverer / clearer than i'm suggesting?

Reply:
Not needed anymore